lib/index.js

```diff
@@ -201,7 +216,11 @@ Mixpanel.prototype.track = function(track) {
   }
 
   // set people properties if present in context.mixpanel.peopleProperties
-  if (!is.empty(peopleProps)) {
+  if (!is.empty(peopleProps) && people) {
+    // add Mixpanel's special props into peopleProps Obj
+    for (var prop in props) {
```

Comment:
we couldn't use `track.properties(traitAliases)` for aliasing off of track because username falls back to user.id() which we don't want so I had to explicitly map the properties.